install-dependencies.sh

```diff
@@ -71,6 +71,7 @@ fedora_deps=(
   golang
   hwloc-devel
   krb5-devel
+  libarrow-devel
```

Comment:
The alternative is to not build it in the cmake oss. It's not worth the trouble to improve this since we're going to move the OSS build to bazel soon-ish.

Reply:
And possibly longer term we will have to write a custom library for this